src/test/sigrt.c

```diff
@@ -15,7 +15,7 @@ static void handle_sigrt(int sig) {
   ++num_signals_caught;
 }
 
-static void my_raise(int sig) {
+static void __attribute__ ((noinline)) my_raise(int sig) {
```

Comment:
There's nothing wrong with this change, but I would expect that if the tests are compiled with inlining then lots of other things would break. Is that not the case?